src/siteconfig/models.py

```diff
@@ -186,6 +186,13 @@ class SiteConfig(models.Model):
             Check this if you want to cap marks at 100%. This setting is only relevent if you are using mark percentages."
     )
 
+    simplified_course_registration = models.BooleanField(
+        verbose_name="Simple course registration", default=False,
```

Comment:
Form shows the wrong name for the field, please remove the verbose name so it uses the field name, which is the name we want.

Reply:
Verbose name removed!